Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/SubmissionResultPositiveEfgsConsentFragment.kt

```diff
@@ -0,0 +1,220 @@
+package de.rki.coronawarnapp.ui.submission
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.accessibility.AccessibilityEvent
+import androidx.fragment.app.activityViewModels
+import androidx.lifecycle.Observer
+import androidx.navigation.fragment.findNavController
+import com.google.android.gms.nearby.exposurenotification.TemporaryExposureKey
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.FragmentSubmissionPositiveEfgsConsentBinding
+import de.rki.coronawarnapp.exception.http.BadRequestException
+import de.rki.coronawarnapp.exception.http.CwaClientError
+import de.rki.coronawarnapp.exception.http.CwaServerError
+import de.rki.coronawarnapp.exception.http.ForbiddenException
+import de.rki.coronawarnapp.nearby.InternalExposureNotificationPermissionHelper
+import de.rki.coronawarnapp.ui.doNavigate
+import de.rki.coronawarnapp.ui.viewmodel.EfgsConsentViewModel
+import de.rki.coronawarnapp.ui.viewmodel.SubmissionViewModel
+import de.rki.coronawarnapp.ui.viewmodel.TracingViewModel
+import de.rki.coronawarnapp.util.DialogHelper
+import de.rki.coronawarnapp.util.IGNORE_CHANGE_TAG
+import de.rki.coronawarnapp.util.observeEvent
+
+class SubmissionResultPositiveEfgsConsentFragment : Fragment(),
+    InternalExposureNotificationPermissionHelper.Callback {
+
+    companion object {
+        private val TAG: String? = SubmissionResultPositiveEfgsConsentFragment::class.simpleName
+    }
+
+    private val efgsViewModel: EfgsConsentViewModel by activityViewModels()
```

Comment:
I'm personally not a big fan of using aberrations in the coding. Could you change this?

Reply:
Would it be descriptive enough if I changed the abbreviation to EuropeanConsentViewModel?